src/database/MongoDatabase.js

```diff
@@ -14,6 +14,7 @@ const HuntRepository = require('./repositories/HuntRepository');
 const RelationshipRepository = require('./repositories/RelationshipRepository');
 const BlacklistRepository = require('./repositories/BlacklistRepository');
 const TopRepository = require('./repositories/TopRepository');
+const { default: GiveRepository } = require('./repositories/GiveRepository');
```

Comment:
Well, vscode auto-completes it, and i use it some times, but it doesn't work. In production, even in dev mode, the `{ default: Var }` require doesn't work at all. Change it to 
```suggestion
const GiveRepository = require('./repositories/GiveRepository');
```

Reply:
I really imported with autocomplete hahahuuh